addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -2,49 +2,54 @@
 <templates xml:space="preserve">
     <t t-name="web.PropertyDefinition" owl="1">
         <div class="o_field_property_definition o_field_widget d-block" t-ref="propertyDefinition">
-            <div class="o_field_property_definition_header d-flex flex-row align-items-center justify-content-between m-3">
-                <h6 t-if="props.readonly" t-out="state.propertyDefinition.string" class="text-truncate"/>
-                <input
-                    t-else=""
-                    class="o_input px-2 me-3"
-                    type="text"
-                    required=""
-                    placeholder="Property Name"
-                    t-model="state.propertyDefinition.string"
-                    t-on-change="onPropertyLabelChange"
-                    t-on-keypress="onPropertyLabelKeypress"
-                />
-                <div class="d-flex flex-row me-3">
+            <div class="d-flex justify-content-between px-3 border-bottom">
+                <div class="border-end py-1">
                     <button
-                        t-attf-class="btn btn-link p-0 m-0 me-2 {{this.isFirst ? 'disabled' : ''}}"
-                        t-on-click="() => this.onPropertyMove('up')">
-                        <i class="fa fa-chevron-up"/>
+                        t-attf-class="btn btn-link ps-0 {{this.isFirst ? 'disabled' : ''}}"
+                        t-on-click="() => this.onPropertyMove('up')" title="Move this Property up">
+                        <i class="fa fa-chevron-up ms-2"/>
                     </button>
                     <button
-                        t-attf-class="btn btn-link p-0 m-0 {{this.isLast ? 'disabled' : ''}}"
-                        t-on-click="() => this.onPropertyMove('down')">
-                        <i class="fa fa-chevron-down"/>
-                    </button>
-                    <button
-                        t-if="!props.readonly &amp;&amp; props.canChangeDefinition"
-                        class="btn text-danger p-0 float-end o_field_property_definition_delete ms-3"
-                        title="Remove Property"
-                        t-on-click="props.onDelete">
-                        <i class="fa fa-trash"/> Delete
+                        t-attf-class="btn btn-link pe-3 {{this.isLast ? 'disabled' : ''}}"
+                        t-on-click="() => this.onPropertyMove('down')" title="Move this Property down">
+                        <i class="fa fa-chevron-down ms-2"/>
                     </button>
                 </div>
+                <button
+                    t-if="!props.readonly &amp;&amp; props.canChangeDefinition"
+                    class="btn pe-0 text-danger float-end o_field_property_definition_delete"
+                    title="Remove Property"
+                    t-on-click="props.onDelete">
+                    <i class="fa fa-trash"/> Delete
+                </button>
             </div>
-            <table class="m-3">
```

Comment:
Replacing  `table` with `div` is making fail several JS tests.. are you sure that this change is necessary?